src/lib/tracer.js

```diff
@@ -1,8 +1,13 @@
 import Tracer from "datadog-tracer"
+import config from "config"
 import { forIn, has } from "lodash"
 import { getNamedType, GraphQLObjectType } from "graphql"
 import * as introspectionQuery from "graphql/type/introspection"
 
+const { DD_TRACER_SERVICE_NAME, DD_TRACER_HOSTNAME } = config
```

Comment:
Nice, I like this pattern (we can/should switch all uses of `process.env` to be this).

Reply:
Yep just refactoring all that now...